package.json

```
@@ -52,18 +51,26 @@
   },
   "homepage": "https://github.com/comunica/sparqlee#readme",
   "devDependencies": {
+    "@rubensworks/eslint-config": "^1.0.1",
     "@types/benchmark": "^2.0.0",
     "@types/jest": "^26.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.29.0",
+    "@typescript-eslint/parser": "^4.29.0",
     "asynciterator": "^3.0.0",
     "benchmark": "^2.1.4",
     "coveralls": "^3.0.2",
+    "eslint": "^7.32.0",
+    "eslint-config-es": "^3.30.7",
```

```suggestion
    "eslint-config-es": "3.30.7",
```
We should pin this one, because I've noticed that they don't really follow semantic versioning. (we also do this in comunica)